blackbox/docs/appendices/release-notes/3.0.0.rst

```diff
@@ -0,0 +1,219 @@
+  query string which contains multiple queries delimited by semicolons.
+
+- Added a new tab for views in the Admin UI which lists available views and
+  their properties.
+
+- Multiple Admin UI improvements.
+
+- Updated the bundled CrateDB Shell (``crash``) to version ``0.24.0`` which
+  adds support for default schema for connections.
+
+- Fully qualified column names including a schema name will no longer match on
+  table aliases.
+
+- The default user if enterprise is disabled changed from ``null`` to
+  ``crate``. This causes entries in ``sys.jobs`` to show up with ``crate`` as
+  username. Functions like ``user`` will also return ``crate`` if enterprise is
+  enabled but the user module is not available.
+
+- Display the node information (name and id) of jobs in the ``sys.jobs`` table.
+
+- Added support for ordering on analysed or partitioned columns.
+
+- Changed the primary key constraints of the information schema tables
+  ``table_constraints``, ``referential_constraints``, ``table_partitions``,
+  ``key_column_usage``, ``columns``, and ``tables`` to be SQL compliant.
+
+- Fixed path of logfile name in rotation pattern in ``log4j2.properties``. It
+  now writes into the correct logging directory instead of the parent
+  directory.
+
+- Added support for views which can be created using the new ``CREATE VIEW``
+  statement and dropped using the ``DROP VIEW`` statement. Views are listed in
+  ``information_schema.views`` and they show up in
+  ``information_schema.tables`` as well as ``information_schema.columns``.
+
+- Enterprise: Added the VIEW privilege class which can be used to grant/deny
+  access to views.
+
+- Added support for ``INSERT INTO ... ON CONFLICT DO NOTHING``. The statement
+  ignores insert values which would cause duplicate keys.
+
+- Arrays can now contain mixed types if they're safely convertible. JSON
+  libraries tend to encode values like ``[ 0.0, 1.2]`` as ``[ 0, 1.2 ]``, this
+  caused an error because of the strict type match we enforced before.
+
+- DEPRECATED: The ``ON DUPLICATE KEY UPDATE`` clause has been deprecated in
+  favor of the ``ON CONFLICT DO UPDATE SET`` clause.
+
+- Added support for ON CONFLICT clause in insert statements.  ``INSERT INTO ...
+  ON CONFLICT (pk_col) DO UPDATE SET col = val`` is identical to ``INSERT INTO
+  ... ON DUPLICATE KEY UPDATE col = val``.  The special EXCLUDED table can be
+  used to refer to the insert values: ``INSERT INTO ... ON CONFLICT (pk_col) DO
+  UPDATE SET col = EXCLUDED.col``
+
+- Added support for ``DEALLOCATE`` statement which is used by certain Postgres
+  Wire Protocol clients (e.g. libpq) to deallocate a prepared statement and
+  release its resources.
+
+- Implemented ``constraint_schema`` and ``table_schema`` in
+  ``information_schema.key_column_usage`` correctly and documented the full
+  table schema.
+
+- Implemented the ``Block Hash Join`` algorithm which is now used for
```

Comment:
I don't think the name of the algorithm should be formatted like inline code. Same for "Equi-Joins". They are just regular terms.